code/lib/cli/src/automigrate/index.ts

```diff
@@ -92,6 +93,8 @@ export const doAutomigrate = async (options: AutofixOptionsFromCLI) => {
     configDir,
     isUpgrade: false,
   });
+
+  await doctor({ configDir, packageManager: options.packageManager });
```

Comment:
Don't we also need to remove one of the existing automigrations that overlaps with `doctor`?

Reply:
yes, but that's already done